src/components/sound.js

```diff
@@ -66,7 +66,12 @@ module.exports.Component = registerComponent('sound', {
 
   remove: function () {
     this.el.removeObject3D('sound');
-    this.sound.disconnect();
+    try {
```

Comment:
Nothing obvious in [THREE.Audio](https://github.com/mrdoob/three.js/blob/master/src/audio/Audio.js). Reaching into the object a bit, `this.sound.source.context.state === 'running'` looked promising but it's still throwing in that state. There's probably an event or something but i can't find much in the AudioNode spec. :neutral_face: